src/main/app/controller/GetController.ts

```diff
@@ -8,7 +8,7 @@ export class GetController {
 
   constructor(
     protected readonly name: string,
-    protected readonly content: Record<string, any>
+    protected readonly content: Record<string, Record<string, unknown>>
```

Comment:
i dont think this is true, content object is mostly property and string value with the expection of erros hence i see why its any but i dont think its an object with values being objects.

Reply:
I double checked the usage of `content` and it does seem to be a language object of objects such as:

```js
const en = {
  question: 'Has your marriage broken down irretrievably (it can’t be saved)?',
  line1: 'You can only get a divorce on the ground that your marriage has broken down irretrievably.',
  yes: 'Yes',
  no: 'No',
  marriageBrokenDownNo: 'The only legal ground for divorce in England and Wales is that your marriage has broken down irretrievably.',
  errors: {
    'screenHasMarriageBroken': {
      'required': 'Select yes if your marriage has broken down irretrievably'
    }
  }
};
```

Reply:
oh yh, my bad, i was thinking of the en object itself.

Reply:
Rather than use `Record<string, unknown>` I think we have `AnyObject` but that might be `Record<string, any>` I can't remember